WeakAuras/BuffTrigger2.lua

```diff
@@ -61,12 +61,20 @@ Returns the potential conditions for a trigger
 local tinsert, wipe = table.insert, wipe
 local pairs, next, type = pairs, next, type
 
+local LCD
```

Comment:
I'm going to trust that you know how to use this library. so I have skipped over changes using this library ;)